lib/top_hangman/game.rb

```diff
@@ -9,15 +9,15 @@ class Game
 
     attr_reader :current_round, :rounds_history, :state
 
-    def initialize
-      @current_round = nil
-      @rounds_history = []
+    def initialize(current_round = nil, rounds_history = [])
+      @current_round = current_round
+      @rounds_history = rounds_history
       @state = nil
```

Comment:
don't you need to load the `@state` from the file too?

Reply:
no, when we start the game at IO, it assign the running state to the Game instance